### DISCUSSION TITLE
Unable to connect to EKS cluster

### DISCUSSION THREAD

**Discussion by Saikartik101** [egovernments/Digit-Core#118] [Local Governance]
Hello! Recently I deployed all the digit services and everything is up and running now I need to modify some services as per my requirements to do that I am searching for where actually the services code available in the cluster, I am trying one approach that is I am trying to connect to ec2 instance cluster using FTP client like FileZilla,winSCP and findout where the services springboot code stored so I can modify them..but the problem here is I am unable to connect to the cluster ec2 instance I tried so many ways but still.. please let me know the solution for this and also let me know that whether I am doing right approach or not for modifying the services or is there anyother approach please let me know...Thank you..

**Comment by talele08** [comment]
If you want to customise any code/module, you will need to create a docker image of the modified code. You can then deploy that docker image on your EKS cluster by using helm chart.
